modules/images/dominant-color/load.php

```diff
@@ -339,3 +320,48 @@ function wp_get_attachment_file_path( $attachment_id, $size = 'medium' ) {
 
 	return $filepath;
 }
+
+/**
+ * Gets the Dominant color for an image attachment.
+ *
+ * @since n.e.x.t
+ *
+ * @param int $attachment_id Attachment ID for image.
+ * @return string|null|WP_Error hex value of dominant color or null if not set.
```

Comment:
See my other comment:
```suggestion
 * @return string|null Hex value of dominant color or null if not set.
```